mhkit/wave/io/cdip.py

```diff
@@ -324,9 +324,13 @@ def request_parse_workflow(
     if not multiyear:
         # Check the cache first
         hash_params = f"{station_number}-{parameters}-{start_date}-{end_date}"
-        data = handle_caching(hash_params, cache_dir)
+        data, _, _ = handle_caching(
+            hash_params,
+            cache_dir,
+            cache_content={"data": None, "metadata": None, "write_json": None},
+        )
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args